app/views/StatusView/index.tsx

```diff
@@ -39,24 +41,24 @@ const STATUS: IStatus[] = [
 	},
 	{
 		id: 'offline',
-		name: 'Invisible'
```

Comment:
If it's not going to be used anymore, it needs to be removed from i18n

Reply:
removed.